how-to/use-theming/client/src/apps.ts

```diff
@@ -0,0 +1,91 @@
+import { App } from "@openfin/workspace";
+
+export async function getApps(): Promise<App[]> {
+	return [experoApp, notificationStudio, processManager, developerContent];
+}
+
+export const experoApp: App = {
```

Comment:
I would swap out the expero view with the theme builder view. It also might be worth seeing if you can add the listener for when the app is already running and see if you schedule a restart and quit if it carries across the last theme passed?

Reply:
Yeah I just copied the apps from home basic starter, I will swap, and see if I can perform the restart

Reply:
Updated, it doesn't retain command line on a restart, so it does temp storage in localStorage and then removes it.